scripts/wrapper/withdraw.js

```diff
@@ -74,15 +74,16 @@ module.exports = function (web3, artifacts) {
         const token = tokenBracketPairs[index][1]
         const bracketAddress = tokenBracketPairs[index][0]
         const usdValue = await amountUSDValue(amount, token, globalPriceStorage)
-        if (usdValue.gte(ONE)) {
+        log(`Request for ${token.symbol} on bracket ${bracketAddress} USD value: ${usdValue}$`)
```

Comment:
@andre-meyer same question here: You actually don't need this code change, right? Or is it because the price feed on rinkeby is unreliable?

Reply:
This change is necessary for testing to see if a withdraw can even work - It can be removed here, but I suggest we move this check out of the withdraw logic when the withdraw logic is used like an API, because the implementation should have control over these checks.

Reply:
Sorry, I am not sure I understand. You can use the logic already like an API, just with this additional check. Why. would you want to do it without the check. 

I am not sure how I can "move this check out of the withdraw logic", since over there is also not the withdraw logic, just the withdraw building and checking logic. 

Should I make the ONE in the usdValue.gte(ONE) as a parameter and then you can use it as an API with an arbitrary value?  LIke that:
ttps://github.com/gnosis/dex-liquidity-provision/pull/new/movingBalanceCheck